lesson2

Relaxiy opened the pull request.

Relaxiy committed:
lesson2

Relaxiy committed:
lesson2

damalikov commented on src/lesson2/Task11.java:
```diff
@@ -0,0 +1,8 @@
+public class Task11 {
+    public static void main(String[] args) {
+        int T = 36600;
+        System.out.println("H" + T / 3600 + "ч M" + T / 60 + "мин S" + T + "с");
```
Задача состоит в том, чтобы указать сколько в этом количестве секунд часов, минут, секунд. 
например 125 секунд должно вывести 0ч. 2м. 5с.

damalikov commented on src/lesson2/Task14.java:
```diff
@@ -0,0 +1,24 @@
+                double b = inB.nextDouble();
+                double c = inC.nextDouble();
+                System.out.println(func(a, b, c));
+                break;
```
Как я упоминал на последнем занятии break использовать запрещено за исключеним, когда мы его используем в switch.

damalikov commented on src/lesson2/Task14.java:
```diff
@@ -0,0 +1,24 @@
+    }
+
+    public static void main(String[] args) {
+        while(true) {
```
Не очень хорошее условие для цикла. Можно получить бесконечный цикл по итогу. Вообще в этой задче циклы в принципе не подрозумивались

damalikov commented on src/lesson2/Task15.java:
```diff
@@ -0,0 +1,28 @@
+
+public class Task15 {
+    public static double func(double a, double b, double c) {
+        double rez = (b + Math.sqrt(Math.pow(b, 2) + 4 * a * c)) / (2 * a) - a * 3 * c + Math.pow(b, -3);
```
можо вернуть это выражение без объявления переменной

damalikov commented on src/lesson2/Task15.java:
```diff
@@ -0,0 +1,28 @@
+    }
+
+    public static void main(String[] args) {
+        while(true) {
```
Смотри комментарии к Task14

Relaxiy replied:
Исправил все задания в двух ветках

Relaxiy committed:
package lesson3;

import java.util.Scanner;

public class Task1 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        int[] a = new int[n];
        a[0] = 1;
        a[1] = 1;
        System.out.println(a[0]);
        System.out.println(a[1]);

        for(int i = 2; i < a.length; i++) {
            a[i] = a[i - 1] + a[i - 2];
            System.out.println(a[i]);
        }

    }
}
